### PR TITLE
Cypress/E2E:Fixed cypress test cases

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary
fixed cypress test cases

#### Release Note
```
NONE
```

### CODE REVIEW

**Review by saturninoabril — APPROVED** [review]
Awesome, thanks @furqanmlk!

File: e2e/cypress/integration/search/post_search_display_spec.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe('Search', () => {
 
         // # Click on searchbox
         cy.get('#searchbarContainer').should('be.visible').within(() => {
-            cy.get('#searchBox').click();
+            cy.get('#searchBox').click({force: true});
```
Comment (saturninoabril) [inline e2e/cypress/integration/search/post_search_display_spec.js:55]: Looks like this one still failed - https://automation-dashboard.vercel.app/cycles/a9e52661-f080-4d3a-9ba6-3dc197a0efa4

**Review by Phrynobatrachus — APPROVED** [review]

**Pull request merged**